### PR TITLE
Ramulator event log reader update

### PR DESCRIPTION
**Pull request by zduey**

### CODE REVIEW

File: cmlreaders/path_finder.py [hunk 1]
```diff
@@ -55,9 +55,9 @@ def __init__(self, subject: Optional[str] = None,
         self.experiment = experiment
         self.session = session
 
-        # Stringify localization and montage
-        self.localization = str(localization)
-        self.montage = str(montage)
+        # Stringify localization and montage only if they are given
+        self.localization = (str(localization) if localization is not None else None)
+        self.montage = (str(montage) if montage is not None else None)
```
Comment (mivade) [inline cmlreaders/path_finder.py]: These parentheses are unnecessary

Comment (mivade) [reply cmlreaders/path_finder.py]: Also, shouldn't we default to 0 for these rather than None?

File: cmlreaders/test/test_path_finder.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ def all_files_subject(rhino_root):
 @pytest.fixture
 def ramulator_files_finder(rhino_root):
     finder = PathFinder('R1354E', rootdir=rhino_root, experiment='PS4_FR5',
-                        session=0)
+                        session='0')
```
Comment (mivade) [inline cmlreaders/test/test_path_finder.py]: We should really be using ints here and converting to strings as needed.

File: cmlreaders/path_finder.py (hunk 1 above)
Comment (zduey) [reply cmlreaders/path_finder.py]: Changed it.

File: cmlreaders/test/test_path_finder.py (hunk 2 above)
Comment (zduey) [reply cmlreaders/test/test_path_finder.py]: Changing these as well.

**Pull request merged**